Fix SSSE3 detection (use SSSE3, not APIC flag).
--- pixman-0.32.0/pixman/pixman-x86.c.orig	2013-11-07 23:43:03.000000000 +0100
+++ pixman-0.32.0/pixman/pixman-x86.c	2013-11-11 09:18:08.030841057 +0100
@@ -170,7 +170,7 @@
 	features |= X86_SSE;
     if (d & (1 << 26))
 	features |= X86_SSE2;
-    if (d & (1 << 9))
+    if (c & (1 << 9))
 	features |= X86_SSSE3;
 
     /* Check for AMD specific features */
